config/types/schema.go

```diff
@@ -113,6 +113,7 @@ type Networkdunit struct {
 
 type Node struct {
 	Filesystem string    `json:"filesystem,omitempty"`
+	Force      bool      `json:"force,omitempty"`
```

Comment:
`Overwrite` seems like it would have a more obvious meaning.

Reply:
renamed